tigerpath/majors_and_certificates/majors/AAS.json

```diff
@@ -182,7 +174,8 @@
                 "SPA 387/AAS 387",
                 "THR 332/AMS 346/GSS 342/LAO 332",
                 "THR 340/AAS 343/CWR 340/HUM 340",
-                "VIS 307/AAS 307"
+                "VIS 307/AAS 307",

```

Comment:
I'm seeing the crosslistings for this course as `DAN 305 / AAS 307 / AMS 310` (on https://aas.princeton.edu/academics/all-courses/fall-2021)